Answers.md

```diff
@@ -0,0 +1,14 @@
+1. Semantic HTML is a nice way of saying "Set the page up so it's easy to change and easy to read."
+We learned a lot about how to position and tag everything correctly in the HTML file and use a 
+seperate CSS files to make it look presentable instead of a mess of content
+
+2. Inline elements are fixed, meaning with a Block element you can manually set the width / height of
+said element.
+
```

Comment:
Hmmm interesting. I think I see what you're getting at. The main point is that 

block - its width is defined by the container it is inside of and no less (unless you specify)
inline - its width is defined by how much space the stuff inside of it is taking, and no more (unless you specify)